app/questionnaire/path_finder.py

```diff
@@ -207,7 +172,11 @@ def get_next_location(self, current_location):
         :param current_location:
         :return: The next location as a dict
         """
-        location_path = self.get_location_path(current_location.group_id, current_location.group_instance)
+        location_path = self.get_routing_path(current_location.group_id, current_location.group_instance)
+        if self.can_reach_summary(location_path):
+            thank_you_location = Location(SchemaHelper.get_last_group_id(self.survey_json), 0, 'thank-you')
+            location_path.append(thank_you_location)
```

Comment:
Do we need thank-you on the location_path. chatting so @ajmaddaford maybe the submit answers should just be coded to redirect there on success?